Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ruby:2.2.1
+
+ENV APP_HOME /thredded
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+ADD . $APP_HOME
+
+RUN apt-get update -qq && \
```

Comment:
You, sir, are a font of knowledge and wisdom.  Should have a revision of this up in the next few hours after I play with it a little bit